graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlService.java

```diff
@@ -42,6 +42,6 @@ static GraphqlServiceBuilder builder() {
      */
     static GraphqlService of(GraphQL graphQL) {
         return new DefaultGraphqlService(graphQL, ctx -> new DataLoaderRegistry(), false,
-                                         GraphqlErrorHandler.of());
+                                         GraphqlErrorHandler.of(), true);
```

Comment:
The default behavior of the builder and factory should match (either opt-in or opt-out by default).

I'm not strong on either default and think it can go either way.

```suggestion
                                         GraphqlErrorHandler.of(), false);
```

Reply:
Done